configure.ac

```diff
@@ -1,6 +1,6 @@
 dnl Process this file with autoconf to produce a configure script.
 
-AC_PREREQ([2.69])
+AC_PREREQ([2.72])
```

Comment:
Do we really need to update the AC requirement to 2.72?
That means we no longer build out-of-the-box on rhel8/9 and many other not-bleeding-edge systems.

Everything else looks good.

Reply:
Reverted this.